### ISSUE TITLE
DelMoral installation issue with unicode

### ISSUE THREAD

**Issue by ethanwhite** [weecology/retriever#535]
Python 2

```
ethan@croryx:~/RetrieverCode (master)$ retriever install csv DelMoral2010
=> Installing Vegetation plots - del Moral, 2010
Downloading MSH_SPECIES_PLOT_YEAR.csv...
Downloading MSH_SPECIES_DESCRIPTORS.csv...lot_year.csv: 148155 / 148155
'utf8' codec can't decode byte 0xf6 in position 51: invalid start byte
Done!
```

Python 3

```
(retrieverpy3)ethan@croryx:~/RetrieverCode (master)$ retriever install csv DelMoral2010
=> Installing Vegetation plots - del Moral, 2010
Downloading MSH_SPECIES_PLOT_YEAR.csv...
Downloading MSH_STRUCTURE_PLOT_YEAR.csv...lot_year.csv: 148155 / 148155
Downloading MSH_SPECIES_DESCRIPTORS.csv..._plot_year.csv: 1743 / 1743
'utf-8' codec can't decode byte 0xf6 in position 1997: invalid start byte
Done!
```

@goelakash - I'm guessing this is related to some of the unicode related changes for Python 3?

**Comment by goelakash** [comment]
I'm looking into this.

**Comment by goelakash** [comment]
@ethanwhite The bug is serious right now. A lot of installs are failing. Should I revert the encoding change until I find a fix?

**Comment by ethanwhite** [comment]
Let's hope into chat and discuss for a few minutes if you're available.

**Comment by ethanwhite** [comment]
Fixed by f4dba513c805771d9ab1efa6be1b4bb0fd0d9e3c.
